src/flb_routes_mask.c

```diff
@@ -39,9 +47,11 @@ int flb_routes_mask_set_by_tag(uint64_t *routes_mask,
     if (!in) {
         return 0;
     }
-
+    uint64_t *tmp;
+    tmp = flb_realloc(routes_mask, sizeof(uint64_t) * routes_arr_size);
```

Comment:
Even though this code is not expected to be included in the final PR I'd like to point out that : 

1. Variables should be defined at the beginning of the function as per the projects coding style (which is a requirement to have the PR merged)
2. The result of memory allocation or reallocation functions has to be handled appropriately.

Reply:
Thanks for the pointers